R/RunDiagnostics.R

```diff
@@ -461,34 +479,45 @@ runCohortDiagnostics <- function(packageName = NULL,
                                          cohortTable = cohortTable,
```

Comment:
I don't know if it something weird with GitHub, but it seems these 3 lines should be deleted?

Reply:
i cant see it on local copy....but i will make some style changes to removes some empty lines